addons/scratchblocks/userscript.js

```diff
@@ -68,14 +60,14 @@ export default async function ({ addon, msg }) {
       parse: scratchblocks.parse,
       render: scratchblocks.render,
       document: options.doc || document,
+      scale: 0.75,
```

Comment:
```suggestion
      scale: 0.675,
```
The default scale of blocks in the Scratch editor is 0.675.